src/include/block.h

```diff
@@ -522,35 +522,57 @@ struct __wt_block_disagg {
  *	The disaggregated block manager custom header
  */
 struct __wt_block_disagg_header {
+#define WT_BLOCK_DISAGG_MAGIC_BASE 0xdb
+#define WT_BLOCK_DISAGG_MAGIC_DELTA 0xdd
+    uint8_t magic;              /* 00: magic byte, one of the values above */
+    uint8_t version;            /* 01: version of writer */
+    uint8_t compatible_version; /* 02: minimum version of reader */
```

Comment:
Unused - sorry if I'm overthinking this or re-prosecuting a design discussion that was had out of band.

Reply:
This is from a discussion with @pmacko86.  The idea is that the writer knows what readers can read this cleanly.  For example, we bump from version 2 to version 3, but we've done it in a way that version 2 can still read it without recompiling.  So we set version=3, compatible_version=2.  The reader is responsible for checking the compatible _version.  I'll put in that check now.